modules/cadentApertureMXBidAdapter.md

```diff
@@ -0,0 +1,62 @@
+# Overview
+
+```
+Module Name:  Cadent Aperture MX Adapter
+Module Type:  Bidder Adapter
+Maintainer: git@emxdigital.com
```

Comment:
is this a working email? please update if not

Reply:
It is working as-is, but we will be going through and updating email addresses with a subsequent PR shortly.